doc/mf6io/extended_modflow.tex

```diff
@@ -4,6 +4,14 @@
 
 This sections describes input files that are only available with Extended \mf.  For more information on using Extended \mf, refer to the \href{https://github.com/MODFLOW-ORG/modflow6/wiki}{MODFLOW 6 Wiki} on the version-controlled \href{https://github.com/MODFLOW-ORG/modflow6}{MODFLOW 6 repository}.
 
+\newpage
+\subsection{NetCDF in Extended \mf}
+Extended \mf supports reading and writing NetCDF (Network Common Data Form) model gridded data in two formats, structured (DIS) and UGRID layered mesh (DIS, DISV). To write a NetCDF file, configure a model name file NetCDF output file option (NETCDF\_STRUCTURED or NETCDF\_MESH2D) and then update individual package input files with the EXPORT\_ARRAY\_NETCDF option. Any number of supported packages for the model can contain this option. Supported packages contain a griddata block and are integrated with the Input Data Processor (see ``Processing of Program Input'' section). Simulation calculations do not need to run in order to generate the model NetCDF file, i.e. it may make sense to generate the file in \mf validate mode (see ``Running a Simulation'' section) as a way of converting existing ASCII griddata to NetCDF. The following example shows a model name file that has configured NetCDF export for UGRID layered mesh with two packages, DIS and IC, contributing gridded data to that file:
```

Comment:
suggest maybe rephrasing to something like

"Extended \mf supports reading and writing gridded array variables to and from NetCDF (Network Common Data Form) files in two formats ... To write an output variable to a NetCDF file, configure a NetCDF output file option (NETCDF\_STRUCTURED or NETCDF\_MESH2D) in a model name file and then configure the EXPORT\_ARRAY\_NETCDF option in a corresponding package input file."

Also, is it that "Supported packages **must** contain a griddata block" to be integrated with IDM?